src/Microsoft.TestPlatform.CrossPlatEngine/EventHandlers/TestRequestHandler.cs

```diff
@@ -72,6 +78,9 @@ public TestRequestHandler() : this(JsonDataSerializer.Instance, new Communicatio
         _onLaunchAdapterProcessWithDebuggerAttachedAckReceived = onLaunchAdapterProcessWithDebuggerAttachedAckReceived;
         _onAttachDebuggerAckRecieved = onAttachDebuggerAckRecieved;
         _jobQueue = jobQueue;
+
+        _fileHelper = new FileHelper();
+        _pathConverter = NullPathConverter.Instance;
     }
 
     protected TestRequestHandler(IDataSerializer dataSerializer, ICommunicationEndpointFactory communicationEndpointFactory)
```

Comment:
Maybe out of this PR, but could this ctor call the one above?

Reply:
I looked into that, but the other ctor does work that can be written in parameters, but it does not look nice. But we could probably refactor it to chain these ctors.